packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -33,17 +35,58 @@ async function updatePackageDependency(
     process.exit(1);
   }
   try {
+    const dependencyType = devDependency ? "devDependencies" : "dependencies";
```

Comment:
I don't think we can assume where they've installed it. Just look in both places. Same for the remove.

Reply:
We can look in both places for upgrade or remove, but if we are installing we have to pick one. I guess this would default to devDependency then?

All of these packages live under devDependencies on the starter